packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/zoomlevel/ZoomUtils.java

```diff
@@ -26,7 +26,7 @@ final class ZoomUtils {
    * @param maximumZoomLevel The maximim supported zoom level.
    * @return An image sensor area based on the supplied zoom settings
    */
-  static Rect computeZoom(
+  static Rect computeZoomRect(
       float zoom, @NonNull Rect sensorArraySize, float minimumZoomLevel, float maximumZoomLevel) {
     final float newZoom = MathUtils.clamp(zoom, minimumZoomLevel, maximumZoomLevel);
```

Comment:
Nit: can this refer to the helper method you wrote below since they are the same?